sdk/nodejs/resource.ts

```diff
@@ -48,7 +94,7 @@ export abstract class Resource {
     /* @internal */ private readonly __providers: Record<string, ProviderResource>;
 
     public static isInstance(obj: any): obj is Resource {
-        return obj && obj.__pulumiResource;
+        return obj && (<Resource>obj).__pulumiResource === true;
```

Comment:
Why was this change necessary? Just for consistency?

Reply:
it helps ensure tat the member shows up as being used.  It's a little thing that bothers me when i'm editing the files.